### ISSUE TITLE
pilot1wrappers is being called in `.Rmd` files

### ISSUE THREAD

**Issue by bms63** [RConsortium/submissions-pilot3-adam#181]
Can these be safely removed or do we need them?

![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/2c6e3dae-0b59-47e6-9ef1-ae98a2e32ddb)

**Comment by robertdevine** [comment]
Yes, and the namespace references changed to {pilot3utils} -all works perfectly in my sandbox. Really a milestone for Pilot3 to have separated out {pilot1wrappers} dependencies. Thx.

**Comment by robertdevine** [comment]
@laxamanaj , @bms63, @kaz462, @SHAESEN2 - the Pilot3 'main' branch is free of {pilot1wrappers} dependencies.
So happy to report this is already done in the earlier work with the Pilot3 Team. Hats off to Pilot3 Team!

What this means is if anyone clones the Pilot3 ADaM repository and on the 'main' branch issues the>renv::status( )
command, no {pilot1wrappers} dependencies exist in the assembly. 

This issue will be closed as 'Completed' tomorrow. Testing in retrospective.

>renv::status()
The following package(s) are used in this project, but are not installed:

	config
	data.table
	devtools
	diffdf
	DT
	lintr
	pilot3utils
	pkgdown
	pkglite
	rprojroot
	sessioninfo
	spelling
	styler
	testthat

Consider installing these packages (for example, using `renv::install()`).
Then, use `renv::snapshot()` to record these packages in the lockfile.
Use `renv::dependencies()` to see where these packages appear to be used.

The following package(s) do not appear to be used in this project:
               _
  BH             [1.81.0-1]
  MASS           [7.3-58.3]
  Matrix         [1.5-3]
  NLP            [0.2-1]
  R6             [2.5.1]
  RColorBrewer   [1.1-3]
  Rcpp           [1.0.10]
  askpass        [1.1]
  assertthat     [0.2.1]
  backports      [1.4.1]
  base64enc      [0.1-3]
  bit            [4.0.5]
  bit64          [4.0.5]
  broom          [1.0.4]
  bslib          [0.4.2]
  cachem         [1.0.7]
  callr          [3.7.3]
  cellranger     [1.1.0]
  checkmate      [2.1.0]
  cli            [3.6.0]
  clipr          [0.8.0]
  colorspace     [2.1-0]
  commonmark     [1.8.1]
  cpp11          [0.4.3]
  crayon         [1.5.2]
  curl           [5.0.0]
  digest         [0.6.31]
  ellipsis       [0.3.2]
  estimability   [1.4.1]
  evaluate       [0.20]
  fansi          [1.0.4]
  farver         [2.1.1]
  fastmap        [1.1.1]
  forcats        [1.0.0]
  formatters     [0.4.0]
  generics       [0.1.3]
  gridExtra      [2.3]
  gtable         [0.3.1]
  highr          [0.10]
  hms            [1.1.2]
  htmltools      [0.5.4]
  httr           [1.4.5]
  isoband        [0.2.7]
  jquerylib      [0.1.4]
  jsonlite       [1.8.4]
  kableExtra     [1.3.4]
  labeling       [0.4.2]
  lattice        [0.20-45]
  lifecycle      [1.0.3]
  magrittr       [2.0.3]
  memoise        [2.0.1]
  mgcv           [1.8-42]
  mime           [0.12]
  munsell        [0.5.0]
  mvtnorm        [1.1-3]
  nlme           [3.1-162]
  numDeriv       [2016.8-1.1]
  openssl        [2.0.6]
  pillar         [1.8.1]
  pkgconfig      [2.0.3]
  prettyunits    [1.1.1]
  processx       [3.8.0]
  progress       [1.2.2]
  ps             [1.7.2]
  purrr          [1.0.1]
  rappdirs       [0.3.3]
  readr          [2.1.4]
  rematch        [1.0.1]
  rlang          [1.1.0]
  rstudioapi     [0.14]
  rvest          [1.0.3]
  sass           [0.4.5]
  scales         [1.2.1]
  selectr        [0.4-2]
  slam           [0.1-50]
  snakecase      [0.11.0]
  stringi        [1.7.12]
  survival       [3.5-5]
  svglite        [2.1.1]
  sys            [3.4.1]
  systemfonts    [1.0.4]
  tidyselect     [1.2.0]
  timechange     [0.2.0]
  tinytex        [0.44]
  tm             [0.7-11]
  tzdb           [0.3.0]
  utf8           [1.2.3]
  vctrs          [0.5.2]
  viridisLite    [0.4.1]
  vroom          [1.6.1]
  webshot        [0.5.4]
  withr          [2.5.0]
  xfun           [0.37]
  xml2           [1.3.3]
  yaml           [2.3.7]

Use `renv::snapshot()` to remove them from the lockfile.

The following package(s) are recorded in the lockfile, but not installed:
             _
  Tplyr        [1.1.0]
  admiral      [0.10.1]
  admiraldev   [0.3.0]
  cowplot      [1.1.1]
  dplyr        [1.1.0]
  emmeans      [1.8.5]
  fs           [1.6.1]
  ggplot2      [3.4.1]
  glue         [1.6.2]
  haven        [2.5.2]
  huxtable     [5.5.2]
  janitor      [2.2.0]
  knitr        [1.42]
  lubridate    [1.9.2]
  metacore     [0.1.2]
  metatools    [0.1.5]
  pharmaRTF    [0.1.4]
  r2rtf        [1.0.1]
  readxl       [1.4.2]
  rmarkdown    [2.20]
  rtables      [0.6.0]
  stringr      [1.5.0]
  tibble       [3.2.0]
  tidyr        [1.3.0]
  visR         [0.3.1]
  xportr       [0.2.0]

Use `renv::restore()` to install these packages.

>

**Comment by bms63** [comment]
I'm very hesitant to touch anything in the lock file.   I think a big learning from this project is to be very systematic with the choices of packages being introduced into the lock file.  

Going to close
